client/src/Components/Chat/ApplicantList.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { useAppSelector } from '../../state/hooks';
+import Applicants from './Applicants';
+import axios from 'axios';
+
+const ApplicantList = () => {
+  const [jobsIPosted, setJobsIPosted] = useState([]);
+  const currUser = useAppSelector((state) => state.userProfile.value);
+
+  const getJobs = async () => {
+    const jobs = await axios.get('/api/jobs/all');
```

Comment:
I could be wrong, but I think this same information is already in state. You could probably avoid an axios call and just useAppSelector(state=>state.job.jobs);